kube/resources/server.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: airbyte-server-svc
+spec:
+  type: NodePort
+  ports:
+    - port: 8001
+      protocol: TCP
+  selector:
+    airbyte: server
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: airbyte-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      airbyte: server
+  template:
+    metadata:
+      labels:
+        airbyte: server
+    spec:
+      containers:
+        - name: airbyte-server-container
+          image: airbyte/server
+          env:
+            - name: AIRBYTE_VERSION
+              value: 0.7.2-alpha # todo
+            - name: CONFIG_ROOT
+              value: /data
+            - name: DATABASE_PASSWORD
+              value: docker
+            - name: DATABASE_URL
+              value: jdbc:postgresql://airbyte-db-svc:5432/airbyte
+            - name: DATABASE_USER
+              value: docker
+            - name: TRACKING_STRATEGY
+              value: segment
+            - name: WAIT_BEFORE_HOSTS
+              value: "5"
+            - name: WAIT_HOSTS
+              value: airbyte-db-svc:5432
+            - name: WAIT_HOSTS_TIMEOUT
+              value: "45"
+          resources:
+            limits:
+              memory: "256Mi"
+              cpu: "500m"
+          ports:
+            - containerPort: 8001
+          volumeMounts:
+            - name: airbyte-volume-data
+              mountPath: /data
+            - name: airbyte-volume-workspace
+              mountPath: /tmp/workspace
+      volumes:
+        - name: airbyte-volume-workspace
+          persistentVolumeClaim:
+            claimName: airbyte-volume-workspace
+        - name: airbyte-volume-data
```

Comment:
should this be config-db? nbd if you prefer data.

Reply:
I'm going to resolve this in a separate PR.